config/settings/base.py

```diff
@@ -522,6 +522,9 @@
 # ------------------------------------------------------------------------------
 EVENTS_QUEUE_URL = env("EVENTS_QUEUE_URL", default=None)
 EVENTS_QUEUE_EXCHANGE_NAME = env("EVENTS_QUEUE_EXCHANGE_NAME", default="amq.fanout")
+EVENTS_QUEUE_POOL_CONNECTIONS_LIMIT = env.int(
+    "EVENTS_QUEUE_POOL_CONNECTIONS_LIMIT", default=50
```

Comment:
We should maintain our current configuration, that's no limit. So I would set a `default=0` and special logic for that case